### DISCUSSION TITLE
请问:lotus转账流程具体API要怎样调用?

### DISCUSSION THREAD

**Discussion by Sun-dong** [filecoin-project/lotus#8309] [General Q&A]
lotus服务已经搭建好了,想请教一下具体流程和手续费具体计算公式,看了好久的官方文档,也没看到个流程,求求了!!

**Comment by TippyFlitsUK** [comment]
Hi @Sun-dong 

Have you seen the information included in the [Filecoin Spec ](https://spec.filecoin.io/#section-systems.filecoin_vm.gas_fee)regarding fees?

Please let me know if you have further questions.

Many thanks!
